extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -437,7 +437,9 @@ private Properties getSchedulerConfigurationProperties(QuartzSupport quartzSuppo
             String dataSource = buildTimeConfig.dataSourceName.orElse("QUARKUS_QUARTZ_DEFAULT_DATASOURCE");
             QuarkusQuartzConnectionPoolProvider.setDataSourceName(dataSource);
             props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".useProperties", "true");
-            props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".misfireThreshold", "60000");
+            props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".misfireThreshold",
+                        "" + quartzSupport.getRuntimeConfig().misfireThreshold);
```

Comment:
```suggestion
            props.put(StdSchedulerFactory.PROP_JOB_STORE_PREFIX + ".misfireThreshold",
                        "" + quartzSupport.getRuntimeConfig().misfireThreshold.toMillis());
```

once the config item has been changed to a `Duration` time, we can convert it to long like this.